solr/core/src/java/org/apache/solr/schema/ManagedIndexSchema.java

```diff
@@ -281,7 +282,7 @@ public static void waitForSchemaZkVersionAgreement(
         }
 
         if (vers == -1) {
-          String coreUrl = concurrentTasks.get(f).coreUrl;
+          String coreUrl = concurrentTasks.get(f).baseUrl;
```

Comment:
should this varialbe *still* be coreUrl?   I guess you could say that this core's url is the base url???